csharp/extractor/Semmle.Extraction.CSharp.Standalone/Extractor.cs

```diff
@@ -146,7 +146,7 @@ public static ExitCode Run(Options options)
             logger.Log(Severity.Info, "Extracting C# in buildless mode");
             using var dependencyManager = new DependencyManager(options.SrcDir, logger);
 
-            if (!dependencyManager.AllSourceFiles.Any())
+            if (!dependencyManager.NonGeneratedSourcesFiles.Any())
```

Comment:
This is a small bugfix to help write a failing buildless integration test. We shouldn't consider generated (implicit usings) files as a source file to decide on success/failure.